examples/test_performance/make_data.py

```diff
@@ -0,0 +1,47 @@
+#This script creates the data necessary to make a graph of how
+#the latency and throughput of an NF chain changes based on length
+#outputs 3 numbers to 'data.txt': [#nfs, TX average, and latency average]
+
+num_nfs = input("Enter the number of nfs: ")
+
+file1 = open("/users/elliott1/openNetVM/examples/output_files/log-speed_tester-1.txt", "r")
```

Comment:
this shouldn't have a hardcoded path (what if I have a different user name?). Better to make it accept a command line argument

Reply:
You're right, I overlooked this. I'm not sure if I've figured out how to pass in command-line arguments successfully